app/src/main/java/com/teamfillin/fillin/presentation/map/MapSearchActivity.kt

```diff
@@ -0,0 +1,99 @@
+package com.teamfillin.fillin.presentation.map
+
+import android.graphics.Color
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.MenuItem
+import android.view.View
+import com.teamfillin.fillin.R
+import com.teamfillin.fillin.core.base.BindingActivity
+import com.teamfillin.fillin.data.ResponseLocationInfo
+import com.teamfillin.fillin.databinding.ActivityMapSearchBinding
+
+class MapSearchActivity : BindingActivity<ActivityMapSearchBinding>(R.layout.activity_map_search) {
+    private val locationAdapter = LocationListAdapter()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        toolbarEvent()
+        editTextIconEvent()
+        setLocationListAdapter()
+    }
+
+    private fun toolbarEvent() {
+        setSupportActionBar(binding.tbTitle)
+        supportActionBar?.run {
+            setDisplayHomeAsUpEnabled(true)
+            setDisplayShowTitleEnabled(false)
+            setHomeAsUpIndicator(R.drawable.ic_back)
+        }
+    }
+
+    private fun editTextIconEvent() {
+        binding.editSearch.addTextChangedListener(object : TextWatcher {
+            override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+            }
+
+            override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+                editTextBlankCheck()
+            }
+
+            override fun afterTextChanged(p0: Editable?) {
+            }
+        })
```

Comment:
```suggestion
        binding.editSearch.doAfterTextChange {
            editTextBlankCheck()
        }
```
Androidx.core에 있는 함수를 활용하는 걸 권장합니다.

Reply:
👍👍